test/cluster/random_failures/cluster_events.py

```diff
@@ -53,6 +56,47 @@ def add_deselected_metadata(fn: Callable[P, T]) -> Callable[P, T]:
     return add_deselected_metadata
 
 
+def feed_rack_seed(seed):
+    global RACK_RANDOM_GENERATOR
+    RACK_RANDOM_GENERATOR = random.Random(seed)
```

Comment:
It would be a bit simpler if we simply moved the TESTS_SHUFFLE_SEED constant to this module. But this way is OK too.